node/src/command.rs

```diff
@@ -78,27 +78,27 @@ pub fn run() -> sc_cli::Result<()> {
     match &cli.subcommand {
         Some(Subcommand::Key(cmd)) => cmd.run(&cli),
         Some(Subcommand::BuildSpec(cmd)) => {
-            let BuildSpecCmd {
-                base,
-                chain_name,
-                chain_id,
-                endowed_accounts,
-                initial_difficulty,
-                ..
-            } = cmd;
-
-            let endowed_accounts = endowed_accounts.clone().unwrap_or_default();
-
-            let spec = chain_spec::custom_config(
-                chain_name,
-                chain_id,
-                endowed_accounts,
-                *initial_difficulty,
-            )?;
-
-            let runner = cli.create_runner(base)?;
-
-            runner.sync_run(|config| base.run(Box::new(spec), config.network))
+            // let BuildSpecCmd {
+            //     base,
+            //     chain_name,
+            //     chain_id,
+            //     endowed_accounts,
+            //     initial_difficulty,
+            //     ..
+            // } = cmd;
+
+            // let endowed_accounts = endowed_accounts.clone().unwrap_or_default();
+
+            // let spec = chain_spec::custom_config(
+            //     chain_name,
+            //     chain_id,
+            //     endowed_accounts,
+            //     *initial_difficulty,
+            // )?;
+
+            let runner = cli.create_runner(&cmd.base)?;
+
+            runner.sync_run(|config| cmd.base.run(config.chain_spec, config.network))
```

Comment:
@fbielejec I had to disable this code to generate the raw spec that I want to launch the chain with in class. Let me start with the very specific problem I encountered, and then zoom out to bigger picture intent.

Specifically, I want to generate the chain spec with `academy-pow build-spec --chain local --raw > spec.json`. This will take the spec that is encoded in Rust in `chain-spec.json` and re-encode it as a raw json spec. Then I make a few edits like adding bootnodes and telemetry and stuff. But with your code here, I _must_ specify endowed accounts on the cli. Actually, the right accounts are already in the Rust code, and I don't want to override them.

So that is basically a little bug, and I'm sure reasonable fixes exist. But I want to take a bigger picture look about why you want those flags in the first place. I think it is a cool idea to let users quickly generate specs like that. If it is just a nice UX feature, I support that.

I'm worried those flags were added, in order to help nodes generate chain specs in your CI bash scripts. If this is the case, I think these flags are basically a way to encode chain spec info in a bash script. But I think it makes more sense to encode that in json or Rust.

LMK what you think. I'm happy to have this CLI feature with a few little tweaks, or happy to guide further on how we might manage chain specs in code.